content/master/software/install.md

```diff
@@ -259,6 +259,7 @@ at the table below.
 | Alpha | `--enable-realtime-compositions` | Enable support for real time compositions. |
 | Alpha | `--enable-ssa-claims` | Enable support for using server-side apply to sync claims with XRs. |
 | Alpha | `--enable-usages` | Enable support for Usages. |
+| Alpha | `--enable-dependency-version-upgrades ` | Enable automatic version upgrades of dependencies when updating packages. |
```

Comment:
Should this flag appear in the 1.17 and 1.16 docs as well?

Reply:
No, we did not cherry-pick the changes to previous releases